README.tpl

```diff
@@ -0,0 +1,7 @@
+# {{crate}} [![crates.io](http://meritbadge.herokuapp.com/{{crate}})](https://crates.io/crates/{{crate}})
```

Comment:
Issue: How are you enforcing that the READMEs don't get out of sync? I consider that an important piece of #363.

Suggestion: Add a task in Makefile.toml that runs all the commands required to re-generate the READMEs, and add a corresponding CI test to ensure that they don't go out of sync.  The CI task can run the Makefile task and check that there are no diffs.  It might be good to put adjacent to the copyright check.